pkg/build/pipelines/split/bin.yaml

```diff
@@ -0,0 +1,38 @@
+name: Split executable files
+
+needs:
+  packages:
+    - busybox
+
+inputs:
+  package:
+    description: |
+      The package to split executable files from
+    required: false
+
+pipeline:
+  - runs: |
+      pkgname="${{context.name}}"
+      pkgnobin="${pkgname%%-bin}"
+      pkgnobindir="${{targets.outdir}}/$pkgnobin"
+
+      if [ -n "${{inputs.package}}" ]; then
+        PACKAGE_DIR="${{targets.outdir}}/${{inputs.package}}"
+      elif [ "$pkgname" != "$pkgnobin" ] && [ -d "$pkgnobindir" ]; then
+        PACKAGE_DIR="$pkgnobindir"
+      else
+        PACKAGE_DIR="${{targets.destdir}}"
+      fi
+
+      if [ "$PACKAGE_DIR" == "${{targets.contextdir}}" ]; then
+        echo "ERROR: Package can not split files from itself!"
+        exit 1
+      fi
+
+      dir=usr/bin
+      if [ -e "$PACKAGE_DIR/$dir" ] || [ -L "$PACKAGE_DIR/$dir" ]; then
+        d="${{targets.contextdir}}/${i%/*}" # dirname $dir
```

Comment:
`$i` came from the loop that I had you drop. sorry. it is not defined here.

Reply:
Apologies. -ERR_EOD_NO_TESTING